app/controllers/items_controller.rb

```diff
@@ -18,6 +21,23 @@ def create
     end
   end
 
+  def destroy
+    @item = current_user.items.find(params[:id])
```

Comment:
@naimaseales Good choice using `current_user`-scoping on line 25 here to ensure items can only be deleted if they belong to the logged-in user.